examples/poc/manifests/gatewayclass.yaml

```diff
@@ -88,28 +90,66 @@ spec:
                     - endpoint:
                         address:
                           socket_address:
-                            address: 34.118.231.147
+                            address: llm-route-ext-proc.llm-gateway.svc.cluster.local
                             port_value: 9002
 ---
 apiVersion: gateway.networking.k8s.io/v1
 kind: GatewayClass
 metadata:
-  name: inference-gateway
+  name: llm-gateway
 spec:
   controllerName: gateway.envoyproxy.io/gatewayclass-controller
   parametersRef:  
     group: gateway.envoyproxy.io
     kind: EnvoyProxy
-    name: custom-proxy-config
-    namespace: envoy-gateway-system
+    name: llm-route-envoy-config
+    namespace: llm-gateway
+
 ---
-apiVersion: gateway.networking.k8s.io/v1
-kind: Gateway
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
gatewayclass as the file name may be misleading if we are putting the ext_proc deployment yaml here

Reply:
The point is that ext_proc is an implementation detail of this specific gateway class

Reply:
A reasonable perspective. 

However, the core of this PoC _is_ the ext-proc implementation, and I don't think can be considered an implementation detail here.

Reply:
how about just call it installation.yaml which just bundles everything, or can break it down to multiple yaml files

Reply:
Either SGTM, thanks!

Reply:
cool, renamed to installation.yaml. And the README explains what's being installed so this should be good!